.github/workflows/test.yml

```diff
@@ -0,0 +1,78 @@
+name: Test
+on: [push]
+jobs:
+  test-linux:
+    name: Shorter linux tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup
+      uses: dpryan79/github-actions/bioconda_utils_setup_conda/@master
+    - name: Install bioconda-utils
+      run: |
+        python setup.py install
+    - name: Build docker container
+      run: |
+        docker build -t bioconda-utils-build-env:latest ./
+        docker history bioconda-utils-build-env:latest
+        docker run --rm -t bioconda-utils-build-env:latest sh -lec 'type -t conda && conda info -a && conda list'
+    - name: Run shorter tests
+      run: |
+        if git diff --name-only origin/master...HEAD | grep -vE ^docs; then
+            py.test --durations=0 test/ -v --tb=native -m 'not long_running'
+        else
+            echo "Skipping pytest - only docs modified"
+        fi
+  test-linux-long:
+    name: Longer linux tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup
+      uses: dpryan79/github-actions/bioconda_utils_setup_conda/@master
+    - name: Install bioconda-utils
+      run: |
+        python setup.py install
+    - name: Build docker container
+      run: |
+        docker build -t bioconda-utils-build-env:latest ./
+        docker history bioconda-utils-build-env:latest
+        docker run --rm -t bioconda-utils-build-env:latest sh -lec 'type -t conda && conda info -a && conda list'
+    - name: Run longer tests
```

Comment:
I'm not familiar with pipeline yet, but is there no way to avoid the repetitive definitions here?

Reply:
Unfortunately not, though at least I got it to about the same length as the circleci tests by off-loading stuff to a different repo. I have to say I prefer the circleci way of writing tests more than how github actions does things, but the latter will likely evolve with time.

Reply:
I should clarify, we can off-load all of this to a github action repository. We should do that for the docker stuff and we could do it for py.test as well, though since that stuff is so short I figured it wasn't worth the hassle of yet another repo.